RELEASE-NOTES.txt

```diff
@@ -1,7 +1,7 @@
 22.6
 -----
 * [**] [internal] Check required WordPress version to set "galleryWithImageBlocks" flag [#20736]
-
+* [**] [internal] [Jetpack-only] Domains selection: Show error message when selecting unsupported domains. [#20786]
```

Comment:
nitpick: I think the `[internal]` is for stuff we want to keep out of the public release notes (see pdcxQM-Km-p2). That doesn't seem to be the case here, although changing it to `*` would lower its priority in the public release notes, which might be what we want. Wdyt?

Reply:
Thanks for sharing the resource! Yes, you're right, I'll update the notes!